framework/integration/testresourceset/resource_set.go

```diff
@@ -0,0 +1,56 @@
+// +build k8srequired
+
+package testresourceset
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/operatorkit/framework/integration/testresourceset/testresource"
+)
+
+type ResourceSetConfig struct {
+	K8sClient kubernetes.Interface
+	Logger    micrologger.Logger
+
+	ProjectName string
+}
+
+func NewResourceSet(config ResourceSetConfig) (*framework.ResourceSet, error) {
```

Comment:
FYI in go you look from the client side to come up with naming decisions. This here is now used like `testresourceset.NewResourceSet`, which stutters. In certain cases `go vet` will complain. This here could just be `New` when the package is descriptive and only holds one specific thing. Same for config and so on. You did it "right" below with the `testresource` package.

Reply:
I fixed this.